### DISCUSSION TITLE
get drip here

### DISCUSSION THREAD

**Discussion by davwat** [CluckCluckChicken/Magnifier#22] [big chungus]
enter drip form $19 fortnite card for only 100 bobux get from mogus drip

**Comment by FunctionalMetatable** [comment]
okay heres big big doge coin for amogus drip
![image](https://user-images.githubusercontent.com/67440879/120066289-e0c66a00-c06d-11eb-89c0-b591eee83124.png)

**Reply by davwat** [reply]
![sus drip](https://user-images.githubusercontent.com/46281840/120066650-5ed74080-c06f-11eb-9aa5-3249b27b5e43.png)

**Reply by davwat** [reply]
there is the drip

**Reply by FunctionalMetatable** [reply]
> there is the drip

yes i rate potatophant drip shop 10000000000000000000000000000000000000000000000/10 great service very fast

**Reply by JackyBorderCollie** [reply]
yess

**Comment by JackyBorderCollie** [comment]
![](https://assets.scratch.mit.edu/4fa10ac36da5f31f6a11b0fd0b18b865.png)

**Comment by JackyBorderCollie** [comment]
![](https://user-images.githubusercontent.com/67440879/120066289-e0c66a00-c06d-11eb-89c0-b591eee83124.png)
